packages-next/types/src/config/hooks.ts

```diff
@@ -92,7 +92,7 @@ type ValidateInputHook<TGeneratedListTypes extends BaseGeneratedListTypes> = (
 ) => Promise<void> | void;
 
 type BeforeOrAfterChangeHook<TGeneratedListTypes extends BaseGeneratedListTypes> = (
-  args: ArgsForCreateOrUpdateOperation<TGeneratedListTypes>
+  args: ArgsForCreateOrUpdateOperation<TGeneratedListTypes> & ({ updatedItem: TGeneratedListTypes['inputs']['update']})
```

Comment:
should this be properly added with `TGeneratedListTypes['inputs']['create']` as well?

Reply:
Yeah, I think this should be:

```
updatedItem: TGeneratedListTypes['inputs']['create'] | TGeneratedListTypes['inputs']['update'];
```

Reply:
done